src/elm/Ui/Section.elm

```diff
@@ -0,0 +1,43 @@
+module Ui.Section exposing (..)
```

Comment:
Only expose the symbols that need exposing. Even if this means everything in the file/module, it's nice as a documentation of "this is the stuff in this module".